docs/source/CONTRIBUTING.rst

```diff
@@ -17,6 +19,8 @@ You can help us by:
 
 Have a look at the `Issues <https://github.com/atmdrops/pydropsonde/issues>`_ and you will find where ``pydropsonde`` needs help. Pick an issue and assign it to yourself, so others know that you are working on that. If you are not sure how to proceed, you can express your interest by commenting on the issue, and someone should help you out.
 
+.. _dev_workflow:
+
 Development Workflow
 --------------------
 
```

Comment:
In the headline following this, it also still says HALO-Drops

Reply:
That too...

Reply:
[Latest commit](https://github.com/atmdrops/pydropsonde/pull/74/commits/440792ebaae6751c07e75a02a9a0624b1b5015f0) fix these now!

Reply:
Nice thanks! I'd give @tmieslinger a chance to have a look, but otherwise I think it's amazing that you fixed all those things